PD-1369 / 25.04 / Pd 1369 update custom app (#3123)
truenas/documentation: content/TruenasApps/_index.md

Some apps proposed by community members might be adopted as official **stable** train apps.
iXsystems maintains official apps for non-Enterprise and community users.

> Click **Settings** on the **Configuration** dropdown menu to open the **Train Settings** screen, then select the desired train(s).

Reviewer: ```suggestion
Click **Settings** on the **Configuration** dropdown menu to open the **Settings** screen, then select the desired train(s).
```

Select the network option, or add additional options to resolve the network connection issues.
{{< /hint >}}

#### Installing NVIDIA Drivers
Beginning in TrueNAS 24.10, NVIDIA drivers are no longer automatically installed. Users must manually install drivers from the TrueNAS UI.

If running TrueNAS 24.10 or higher:
1. Go to **Apps > Installed** and click on the **Configuration**.

2. Click on **Settings** to open the **Settings** configuration screen.

3. Select **Install NVIDIA Drivers**, which is available to users with compatible GPUs.

4. Select **Install NVIDIA Drivers**, and click **Save.**

#### Monitoring for Image Updates

Select **Check for docker image updates** (selected by default) to enable TrueNAS to periodically check for Docker image updates.

> This applies to all Docker images present on the system that may or may not for either catalog or custom applications.

Reviewer: don't user "may" also something is missing in this sentence. incomplete thought?
```suggestion
This applies to all Docker images present on the system for either catalog or custom applications.
```

Author: ah that's what I get for copying over Stavros' explanation from slack

Restart the system then return to the **Resources Configuration** section of the application to see if expected devices are available.
{{< /expand >}}

### Installing Custom Applications
{{< include file="/static/includes/apps/CustomAppIntro.md" >}}

See [Installing Custom Applications]({{< relref "UsingCustomApp.md" >}}) for more information.

## Upgrading Apps

> Apps display a yellow circle with an exclamation point that indicates an upgrade is available, and the **Installed** application screen banner displays an **Update** or **Update All** button when upgrades are available.

Reviewer: ```suggestion
Apps display a yellow circle with an exclamation point to indicate an upgrade is available, and the **Installed** application screen banner displays an **Update** or **Update All** button when upgrades are available.
```
